proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -87,3 +87,45 @@ message SearchResponse {
   // There are no more pages if this is 0.
   uint32 next_page_token = 2;
 }
+
+message SearchUsersRequest {
```

Comment:
I would add the RPCs at the same time

Reply:
can you look at discussion [here](https://github.com/bufbuild/core/issues/2857) to help us decide if this is the direction we want to go?

Reply:
Commented - I would decide on that direction first though before adding request/response messages here